packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/test/Microsoft.Generator.CSharp.ClientModel.Tests.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <ItemGroup>
+    <ProjectReference Include="..\..\Microsoft.Generator.CSharp.ClientModel.Plugin\src\Microsoft.Generator.CSharp.ClientModel.Plugin.csproj" />
```

Comment:
I don't think we want this dependency, any plugin tests should go into its test project?